Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_baking.py

```diff
@@ -347,6 +351,19 @@ def incrementProgress(self, context, message, obj=None, reset=False):
 
         bakeSettings.bake_message = message
 
+    def addRealizeInstances(self, mod):
+        nodes = mod.node_group.nodes
+        outputs = [x for x in nodes if x.type == "GROUP_OUTPUT"]
+
+        for output in outputs:
+            if len(output.inputs) == 0 or len(output.inputs[0].links) == 0:
```

Comment:
I think instead of checking the number of inputs, the 'is_active_output' flag should be used.

Reply:
I tried that but noticed that is_active_input is true for the node that is selected last. So if have 2 outputs nodes and only one has links but I select the empty one, the bake will not include the instances. 

My solution is to simply try and add the realize instance node if possible, that is the output node has at least one input with at least one link.

Reply:
Yeah the selected node should be the active one, that's how it works for materials as well. Would there be any harm to just add the realize instances node before every output?